File: src/main/java/Level1/Ex17.java
package Level1;

import java.util.stream.Stream;

//자릿수 더하기
public class Ex17 {
    public static int solution(int n) {
        int answer = 0;
        int[] intList = Stream.of(String.valueOf(n).split("")).mapToInt(Integer::parseInt).toArray();
        for (int i = 0; i < intList.length; i++) {
            answer += intList[i];
        }
        return answer;
    }

    public static void main(String[] args) {
        int n = 987;
        System.out.println(solution(n));
    }
}

File: src/main/Javascript/Level1/Ex17.js
function solution(n) {
    let answer = 0;
    let nums = [];
    do {
        nums.push(n % 10);
        n = Math.floor(n / 10);
    } while (n > 0);
    console.log(nums);
    for (let i = 0; i < nums.length; i++) {
        answer += nums[i];
    }
    return answer;
}

console.log(solution(123));
console.log(solution(987));